config/initializers/breakers.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+require 'rx/configuration'
+
+# Read the redis config, create a connection and a namespace for breakers
+redis_config = Rails.application.config_for(:redis).freeze
```

Comment:
Why do think it is better to read this config here rather than using the REDIS_CONFIG constant that was already set in redis.rb? Oh... because of the order that they are loaded... hmmm

What if breakers was initialized from redis.rb after those constants were defined?